Add a new XDR use case - Send notifications to a Webhook using playbooks (#1352)
SEKOIA-IO/documentation: docs/getting_started/notifications-Examples.md

!!! info 
    You can’t use the WebHook notification mechanism to push information directly to third parties (such as Slack or Telegram), you have to use an intermediate server. To do so, you can use solutions like IFTTT or a simple HTTP server (see below).

>     There are also playbook templates that can be used to send notifications to Slack or Microsoft Teams directly using Webhooks; See how to [send notifications to a Webhook using a playbook](../xdr/usecases/playbook/notifications_using_playbooks.md) for more information 

Reviewer: ```suggestion
    There are also playbook templates that can be used to send notifications to Slack or Microsoft Teams directly using Webhooks. See how to [send notifications to a Webhook using a playbook](../xdr/usecases/playbook/notifications_using_playbooks.md) for more information.
```

>     There are also playbook templates that can be used to send notifications to Slack or Microsoft Teams directly using Webhooks, see this documentation for more information [Send notifications to a Webhook using a playbook](../xdr/usecases/playbook/notifications_using_playbooks.md)

Reviewer 2: ```suggestion
    There are also playbook templates that can be used to send notifications to Slack or Microsoft Teams directly using Webhooks; See how to [send notifications to a Webhook using a playbook](../xdr/usecases/playbook/notifications_using_playbooks.md) for more information 
```
